pkg/debugapi/router.go

```diff
@@ -68,6 +68,10 @@ func (s *server) setupRouting() {
 	router.Handle("/peers", jsonhttp.MethodHandler{
 		"GET": http.HandlerFunc(s.peersHandler),
 	})
+	router.Handle("/peers/blocklisted", jsonhttp.MethodHandler{
```

Comment:
This endpoint clashes with `"/peers/{address}"` (just bellow it).
While they do use different method, and this is not public API, I still think some other than `"/peers/blocklisted"` should be used (maybe just reverse words). At least to keep the API "clean" and discourage similar practice for the future.

Reply:
Thanks @zbiljic. I renamed the endpoint to `/blocklist`